app/src/main/java/com/limelight/binding/input/ControllerHandler.java

```diff
@@ -1249,13 +1264,18 @@ else if (foundMatchingDevice && !vibrated && prefConfig.vibrateFallbackToDevice)
         }
     }
 
-    public boolean handleButtonUp(KeyEvent event) {
+    public boolean handleButtonUp(KeyEvent event, PreferenceConfiguration prefConfig) {
```

Comment:
`ControllerHandler` has a `PreferenceConfiguration` passed to its constructor. Can you store the `flipFaceButtons` value there rather than passing in `prefConfig` to `handleButtonUp()` and `handleButtonDown()`?

Reply:
Didn't even notice it was already here. Fixed!